ingestion_tools/dataset_configs/template.yaml

```diff
@@ -86,18 +88,23 @@ datasets: REQUIRED
     cell_component: OPTIONAL
       id: OPTIONAL, STRING (GO_ID)
       name: REQUIRED, STRING
+      pydantic__no_name_validation: see annotations.annotation_object.pydantic__no_name_validation
```

Comment:
This seems a little dirty to me that we're adding a field to our **ingestion schema** that toggles ontology verification on/off. What's the advantage to doing it here vs making it a runtime flag like `--enforce-ontologies` to the validation script? Are we trying to enforce ontologies by default and just recording this escape hatch in our config files so it's explicit?

Reply:
I think the idea is that with a flag here, then individual entries can be marked to skip validation, as if we did --enforce-ontologies we could do something similar by specifying a file, but that may not give the same level of granularity.